lib/src/disposable_manager/disposable_manager_vm.dart

```diff
@@ -14,7 +14,7 @@
 
```

Comment:
+1 – I suggested a `_common` suffix but yeah you don't even need that, I'm cool with `disposable_manager.dart`